src/wp-admin/includes/plugin-install.php

```diff
@@ -912,9 +913,10 @@ function install_plugin_information() {
  * }
  * @param bool         $compatible_php   The result of a PHP compatibility check.
  * @param bool         $compatible_wp    The result of a WP compatibility check.
+ * @param string       $context          Optional. The context the button will appear in. Default empty string.
```

Comment:
What do you think about adding what values it accepts?

For example, `sanitize_bookmark_field()` is documented as:

```

Reply:
I like that description. It's very clear what that param accepts. Any other value is not (currently) used.

Reply:
It's useful in that devs / contributors will know that this param only accepts that value. In the future, more values will likely be added.

Reply:
Implemented in [30832d9](https://github.com/WordPress/wordpress-develop/pull/6732/commits/30832d9762e82e5d8725c08262e83a421188ce6f).